package.json

```diff
@@ -6,6 +6,9 @@
   "scripts": {
     "build": "webpack --mode production",
     "start": "webpack --watch --mode development",
+    "link": "cd build && npm link",
+    "link:portal-app": "npm link",
```

Comment:
Can you think about more general title for the script? `link:portal-app` seems to be too specific for just `"npm link",` command.

Reply:
how about `link:vite`  or `link:vite-project ` since its a Vite issue

Reply:
`link:vite-project` sounds better

Reply:
Maybe i should also change `link` to` link:webpack-project` to make them similar? WDYT?

Reply:
I think the best would be to not use `webpack` / `vite` terminology but if there is a known issue with a `vite` thats fine as a fix to menion it. I would expect that the `linking` works the very same way `publish` work.